dre-service/src/main/java/io/cdap/re/YARERulebookHandler.java

```diff
@@ -457,4 +532,26 @@ public void deleteRuleFromRb(HttpServiceRequest request, HttpServiceResponder re
     });
   }
 
+  private NamespacedId getNamespacedId(String namespace, String id) throws IOException, NamespaceNotFoundException {
+    Namespace ns = getNamespace(namespace);
+
+    return new NamespacedId(ns, id);
+  }
+
+  private Namespace getNamespace(String namespace) throws IOException, NamespaceNotFoundException {
+    NamespaceSummary namespaceSummary;
+
+    if (Contexts.SYSTEM.equals(namespace)) {
```

Comment:
nit: it's a little more readable to remove the else clause:

```
if (Contexts.SYSTEM.equals(namespace)) {
  return new Namespace(namespace, 0L);
}

NamespaceSummary namespaceSummary = getContext().getAdmin().getNamespaceSummar(namespace);
...
```

Reply:
Done